dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -205,9 +209,10 @@ def test_meta_nonempty_index():
     assert type(res) is pd.RangeIndex
     assert res.name == idx.name
 
-    idx = pd.Int64Index([1], name="foo")
+    idx = pd.Index([1], name="foo", dtype="int")
     res = meta_nonempty(idx)
-    assert type(res) is pd.Int64Index
+    assert isinstance(res, pd.Index)
```

Comment:
Nitpick: it looks like we've backed off the `is` comparison here in favor of `isinstance`, which is less precise. Maybe `assert type(res) is type(idx)` would be equivalent to what's currently in `main`?

Reply:
yes! Good call. I think I had a different solution originally where this was the best I could do.